baton-maven-plugin/src/main/java/org/technologybrewery/baton/BatonMojo.java

```diff
@@ -230,37 +232,17 @@ protected void loadMigrations() {
                         migrationsStream,
                         tempMigrationsFile.toPath(),
                         StandardCopyOption.REPLACE_EXISTING);
```

Comment:
A: not a big deal, but I think inlining this change creates more cognitive friction to someone reading the code.  It was super clear what was happening before, b/c of the method name.  Now you have to read two lines and interpret on the fly to understand.  Sometimes it is not how many lines, but how easy is it to follow.

A: line 236 should be indented, if this code stays this way.